code/modules/jobs/job_types/warden.dm

```diff
@@ -16,6 +16,8 @@
 
 	outfit = /datum/outfit/job/warden
 
+	alt_titles = list("Brig Watchman", "Brig Superintendent", "Security Lt. Commander", "Security Dispatcher", "Prison Supervisor")
```

Comment:
```suggestion
	alt_titles = list("Brig Watchman", "Brig Superintendent", "Security Staff Sergeant", "Security Dispatcher", "Prison Supervisor")
```

OKAY, Security Staff Sergeant could be used here since Lt Commander may be a little high up there for the warden. My thinking is HoS is the CO while Warden the NCO.